Library/Formula/lcm.rb

```diff
@@ -0,0 +1,27 @@
+class Lcm < Formula
+  desc "libraries and tools for message passing and data marshalling"
+  homepage "https://lcm-proj.github.io/"
+  url "https://github.com/lcm-proj/lcm/releases/download/v1.2.1/lcm-1.2.1.zip"
+  sha256 "55b959409fc1c0cfa705209b679dc4b3043b68f14078e192b65c1b6021c37ea6"
+
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+  depends_on :java => :recommended
+  depends_on :python => :recommended if MacOS.version <= :snow_leopard
+  depends_on :python3 => :optional
+
+  def install
+    ENV["_JAVA_OPTIONS"] = "-Duser.home=#{ENV["HOME"]}"
```

Comment:
Sounds good. I'll try to fit this in over lunch today.

On Mon, Jan 25, 2016, 04:29 Dominyk Tiller <notifications@github.com> wrote:

> In Library/Formula/lcm.rb
> <https://github.com/Homebrew/homebrew/pull/44455#discussion_r50687634>:
>
> > @@ -0,0 +1,27 @@
> > +class Lcm < Formula
> > +  desc "libraries and tools for message passing and data marshalling"
> > +  homepage "https://lcm-proj.github.io/"
> > +  url "https://github.com/lcm-proj/lcm/releases/download/v1.2.1/lcm-1.2.1.zip"
> > +  sha256 "55b959409fc1c0cfa705209b679dc4b3043b68f14078e192b65c1b6021c37ea6"
> > +
> > +  depends_on "pkg-config" => :build
> > +  depends_on "glib"
> > +  depends_on :java => :recommended
> > +  depends_on :python => :recommended if MacOS.version <= :snow_leopard
> > +  depends_on :python3 => :optional
> > +
> > +  def install
> > +    ENV["_JAVA_OPTIONS"] = "-Duser.home=#{ENV["HOME"]}"
>
> You can use ENV.java_cache here now, after you've rebased against an
> up-to-date master.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/44455/files#r50687634>.
>